ivy/functional/backends/torch/experimental/activations.py

```diff
@@ -41,8 +41,10 @@ def relu6(
 
 @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, backend_version)
 def logsigmoid(
-    input: torch.Tensor, /, *, out: Optional[torch.Tensor] = None, complex_mode="jax"
+    input: torch.Tensor, /, *, out: Optional[torch.Tensor] = None
```

Comment:
I think you should add `complex_mode="jax"` back in, the backends should have the same call signatures as the ivy functions